### PR TITLE
setInlineStyle implementation + test

### PR DESCRIPTION
**Pull request by enchev**
Implemented https://github.com/NativeScript/NativeScript/issues/619

### CODE REVIEW

File: ui/core/view-common.ts [hunk 1]
```diff
@@ -923,6 +923,12 @@ export class View extends proxy.ProxyObject implements definition.View {
         return false;
     }
 
+    public setStyle(style: string): void {
+        if (types.isString(style)) {
```
Comment (hshristov) [inline ui/core/view-common.ts]: if not string - I think we should thrown an exception.

File: ui/core/view-common.ts [hunk 2]
```diff
@@ -923,6 +923,12 @@ export class View extends proxy.ProxyObject implements definition.View {
         return false;
     }
 
+    public setStyle(style: string): void {
```
Comment (hshristov) [inline ui/core/view-common.ts]: And name it setInlineStlye to make sure that it will override CSS style.

**Comment by hshristov** [comment]
:+1:

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**